java/KinesisConnectors/src/main/resources/flink-application-properties-dev.json

```diff
@@ -2,15 +2,15 @@
   {
     "PropertyGroupId": "InputStream0",
     "PropertyMap": {
-      "stream.arn": "arn:aws:kinesis:us-east-1:012345678900:stream/ExampleInputStream",
+      "stream.arn": "arn:aws:kinesis:us-east-1:920373029663:stream/ExampleInputStream",
```

Comment:
Be sure not to commit real ARN.
My suggestion is to use a placeholder, like `arn:aws:kinesis:<region>:<account>:stream/ExampleInputStream` because it makes obvious that it has to be changed.

but a dummy ARN like `arn:aws:kinesis:us-east-1:012345678900:stream/ExampleInputStream` might work as well